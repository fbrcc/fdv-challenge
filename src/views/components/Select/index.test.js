import React from 'react';
import { Select } from './index'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
      .create(<Select name="country" countries={[{name:"Argentina", alpha2code:"AR"}, {name:"Brazil", alpha2code:"BR"}]} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });


