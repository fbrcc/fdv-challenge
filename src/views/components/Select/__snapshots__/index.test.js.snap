// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<select
  className="sc-bdVaJa lnCdxW"
  name="country"
  onChange={[Function]}
  value={undefined}
>
  <option
    value=""
  >
    Pa√≠s
  </option>
  <option
    value="Argentina"
  >
    Argentina
  </option>
  <option
    value="Brazil"
  >
    Brazil
  </option>
</select>
`;
